ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/AbstractWorldUpdater.java

```diff
@@ -338,6 +344,16 @@ public UInt256 getOriginalStorageValue(final UInt256 key) {
       return entries;
     }
 
+    @Override
+    public List<BytesValue> getAccountProof() {
+      return account.getAccountProof();
+    }
+
+    @Override
+    public List<BytesValue> getStorageEntry(final Bytes32 key) {
+      return account.getStorageEntry(key);
```

Comment:
I am not confident about this change

Reply:
I think the names `getAccountProof` and `getStorageEntry` confused me as to their purpose.  I think at least `getStorageEntry` should be renamed to `getStorageProof`, in addition to @mbaxter's refactoring requests.